cms/bundles/tests/test_viewsets.py

```diff
@@ -448,6 +448,12 @@ def test_choose_view(self):
         self.assertNotContains(response, self.page_live.get_admin_display_title())
         self.assertNotContains(response, self.page_draft_in_bundle.get_admin_display_title())
 
```

Comment:
I'm wondering if it's worth creating another page in a non-default locale and confirm that if it's returned? (here and in other tests)
```suggestion
InformationPage.objects.get(locale=Locale.objects.get(language_code="cy"))
```

Reply:
32a4a53 doesn't seem to test that the Welsh draft title is returned. Other than that LGTM